src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -169,7 +169,7 @@ public void RebaseUriVisitor_VisitFileData_PatchesParentUri()
 
             run = rebaseUriVisitor.VisitRun(run);
 
-            run.Files[0].FileLocation.Uri.Should().Be(rootUriBaseId);            
+            run.Files[0].FileLocation.Uri.Should().Be("blah.zip");
```

Comment:
>blah [](start = 55, length = 4)

Let's also assert that `run.Files[0].FileLocation.UriBaseId.Should().Be("SRCROOT")`.